src/components/dialogs/parameters/network-visualizations/map-parameters.tsx

```diff
@@ -36,11 +45,102 @@ export const MapParameters = () => {
             label: '100',
         },
     ];
+    const watchLineFlowColorMode = useWatch({
+        name: `${TabValue.MAP}.${PARAM_LINE_FLOW_COLOR_MODE}`,
+    });
+
+    const onSliderChange = (value: any) => {
```

Comment:
I think here you can use number as type: const onSliderChange = (value: number)

Reply:
Done